indexdata.go

```diff
@@ -84,6 +89,34 @@ type indexData struct {
 	repoListEntry RepoListEntry
 }
 
+type symbolData struct {
+	symContent       []byte
```

Comment:
why are we storing symContent? Isn't that already part of the sections in the file content?

Reply:
symContent is an enum of all the symbol/parent names that ctags returns such as `zoekt.IndexBuilder` or `IndexBuilder`. It's a `[]string`.

I realize that the symbol names probably don't need to be stored in an enum in the index file since it's all unique. We can represent symbol names as `<file idx uint32><start uint32><end uint32>`.